src/apps/ENGINE/internal_functions.cpp

```diff
@@ -1845,7 +1845,7 @@ DATA *COMPILER::BC_CallIntFunction(uint32_t func_code, DATA *&pVResult, uint32_t
 
         CreateMessage(&ms, s_off, 1);
 
-        uint64_t mresult = -1;
+        uint64_t mresult = 0;
```

Comment:
btw, I've just remembered that it was me who initialized `mresult` with -1. It wasn't initialized before and the address sanitizer caught an uninitialized variable access here. Check the commit ec6dee6 - Fix reading uninitialized variable

Reply:
Well...well, maybe it's not related then and safe to merge? But why it was causing issues if it was uninitialized before?

Reply:
I think the engine wasn't properly tested after merging thunderstorm (the initialization was removed in 9fdf420 - Build engine executable)